plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/json/network/Ipam.java

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.docker.client.json.network;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * author Alexander Garagatyi
```

Comment:
I'll add @ but doesn't know what to write as a description since I just copied structure from docker API. So I'll let to do that to someone who knows something meaningful about that object